src/main/java/com/example/handlingformsubmission/PublishTextSMS.java

```diff
@@ -0,0 +1,39 @@
+package com.example.handlingformsubmission;
+
+import org.springframework.stereotype.Component;
+import software.amazon.awssdk.auth.credentials.AwsBasicCredentials;
+import software.amazon.awssdk.auth.credentials.StaticCredentialsProvider;
+import software.amazon.awssdk.regions.Region;
+import software.amazon.awssdk.services.sns.SnsClient;
+import software.amazon.awssdk.services.sns.model.PublishRequest;
+import software.amazon.awssdk.services.sns.model.SnsException;
+
+@Component("PublishTextSMS")
+public class PublishTextSMS {
+
+    public void sendMessage(final String id) {
+        final Region region = Region.US_EAST_1;
+        final AwsBasicCredentials awsCredentials = AwsBasicCredentials.create(
+                "AKIAIOSFODNN7EXAMPLE",
```

Comment:
Recommendation generated by Amazon CodeGuru Reviewer. Leave feedback on this recommendation by replying to the comment or by reacting to the comment using emoji.

It appears your code contains a hardcoded AWS Access Key ID. Hardcoded secrets or credentials can allow attackers to bypass authentication methods and perform malicious actions. We recommend revoking access to resources that use this access key and using IAM roles instead of hardcoded access keys for IAM users. If you must use IAM users, secure their secrets outside your code.

[Learn more about the use of hardcoded credentials](https://docs.aws.amazon.com/general/latest/gr/aws-access-keys-best-practices.html)